hokusai/production.yml

```diff
@@ -159,3 +159,40 @@ spec:
     - 104.16.0.0/12
     - 172.64.0.0/13
     - 131.0.72.0/22
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: force
+    component: web
+    layer: application
+  name: force-web-internal
+  namespace: default
+spec:
+  ports:
+    - port: 5000
+      protocol: TCP
+      name: force-http
+      targetPort: force-http
+  selector:
+    app: force
+    layer: application
+    component: web
+  type: ClusterIP
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: force
+  annotations:
+    nginx.ingress.kubernetes.io/whitelist-source-range: "{{ cloudflareIpSourceRanges|join(',') }}"
```

Comment:
Looks like the current nginx config _is_ custom, but only for the reason of port `5000`. 👍

Reply:
Are we required to use this port? almost everything else uses `8080`.

Reply:
I wondered about it too. Changing it to 8080 ought to be easy, but from grepping through the repo, I see references to port 5000 in quite a bunch of files, so I am not sure. Leaning towards leaving it alone for now.